source/sass/views/dear-internet.scss

```
@@ -0,0 +1,358 @@
+
+    animation-name: float-right;
+    animation-timing-function: ease-in-out;
+    animation-iteration-count: infinite;
+    position: absolute;
+
+    @media (min-width: $breakpoint) {
+      --position-scaler: 20px;
+    }
+
+    @for $i from 1 through 8 {
+      &:nth-child(#{$i}) {
+        animation-duration: #{random($animation-duration) + 15}s;
+
+        @if random() > 0.5 {
+          animation-name: float-left;
+        }
+      }
+    }
+
+    svg.cloud {
+      --width: 140px;
+
+      width: var(--width);
+
+      @media (min-width: $breakpoint) {
+        --width: 200px;
+      }
+
+      &.blue path {
+        fill: $blue;
+      }
+
+      &.light-blue path {
+        fill: $light-blue;
+      }
+
+      &.white path {
+        fill: $white;
+      }
+
+      &.light-red path {
+        fill: $light-red;
+      }
+    }
+  }
+
+  @keyframes float-left {
+    0% {
+      transform: translatey(0);
+    }
+    50% {
+      transform: translateY(20px) translateX(-25px);
+    }
+    100% {
+      transform: translatey(0);
+    }
+  }
+
+  @keyframes float-right {
+    0% {
+      transform: translatey(0);
+    }
+    50% {
+      transform: translateY(-35px) translateX(15px);
+    }
+    100% {
+      transform: translatey(0);
+    }
+  }
+
+  .clouds-wrapper {
+    overflow-x: hidden; // to prevent clouds from going outside of viewport
+
+    .cloud-container {
+      &:nth-child(1) {
+        top: calc(2.5 * var(--position-scaler));
+        left: 5%;
+
+        svg.cloud {
+          transform: scale(0.7) scaleX(-1);
+        }
+      }
+
+      &:nth-child(2) {
+        top: calc(2.2 * var(--position-scaler));
+        left: 80%;
+      }
+
+      &:nth-child(3) {
+        top: calc(12 * var(--position-scaler));
+        left: 10%;
+
+        svg.cloud {
+          transform: scale(1.2) rotateY(180deg);
+        }
+      }
+
+      &:nth-child(4) {
+        top: calc(25 * var(--position-scaler));
+        left: 65%;
+
+        svg.cloud {
+          transform: scale(2);
+        }
+      }
+
+      &:nth-child(5) {
+        top: calc(30 * var(--position-scaler));
+        left: 82%;
+
+        svg.cloud {
+          transform: scale(1.2) rotateY(180deg);
+        }
+      }
+
+      &:nth-child(6) {
+        top: calc(34 * var(--position-scaler));
+        left: -2%;
+
+        svg.cloud {
+          transform: scale(2);
+        }
+      }
+
+      &:nth-child(7) {
+        top: calc(40 * var(--position-scaler));
+        left: 18%;
+
+        svg.cloud {
+          transform: scale(1.5) rotateY(180deg);
+        }
+      }
+
+      &:nth-child(8) {
+        top: calc(46 * var(--position-scaler));
+        left: 58%;
+      }
+    }
+  }
+
+  .love-letter-stamp {
+    margin: 2.5rem 0;
+
+    @media (min-width: $breakpoint) {
+      margin: 5rem 0;
+    }
+
+    img {
+      --width: 80px;
+
+      width: var(--width);
+
+      @media (min-width: $breakpoint) {
+        --width: 128px;
+      }
+    }
+  }
```

a variable makes sense when it's used in multiple places, but this one looks like it's only used for the single `width` rule, so that's better kept "plain":

```suggestion
    img {
      width: 80px;

      @media (min-width: $breakpoint) {
        width: 128px;
      }
    }
  }
```